Refresh.GameServer/Endpoints/Game/DataTypes/Response/GameUserResponse.cs

```diff
@@ -130,6 +131,7 @@ private void FillInExtraData(GameUser old, TokenGame gameVersion, GameDatabaseCo
                 //Fill out LBP2/LBP1 levels
                 goto case TokenGame.LittleBigPlanet2;
             }
+            case TokenGame.BetaBuild:
```

Comment:
Given the beta builds are fully isolated, shouldnt this logic be different than LBP2?